uni/lib/controller/local_storage/app_shared_preferences.dart

```diff
@@ -138,29 +138,31 @@ class AppSharedPreferences {
 
   /// Returns the user's faculties
   static Future<List<String>> getUserFaculties() async {
-    final prefs = await SharedPreferences.getInstance();
-    final storedFaculties = prefs.getStringList(userFaculties);
-    return storedFaculties ?? ['feup'];
+    const storage = FlutterSecureStorage();
+    final storedFaculties = await storage.read(key: userFaculties);
```

Comment:
The faculties are not sensitive, so should be stored on the regular app preferences.

Reply:
changed to app preferences